scripts/in_container/run_provider_yaml_files_check.py

```diff
@@ -483,20 +481,24 @@ def check_extra_link_classes(yaml_files: dict[str, dict]) -> tuple[int, int]:
     return num_extra_links, num_errors
 
 
+@run_check("Checking extra-links belong to package, exist and are classes")
```

Comment:
I noticed the opportunity of extracting the common code here - and added missing check for `executors`